delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphViews.scala

```diff
@@ -339,8 +340,8 @@ final class BlazegraphViews(
       IO.pure(toIndexViewDef(envelope))
     }
 
-  private def toIndexViewDef(envelope: Envelope[BlazegraphViewState]) =
-    envelope.toElem { v => Some(v.project) }.traverse { v =>
+  private def toIndexViewDef(elem: Elem.SuccessElem[BlazegraphViewState]) =
+    elem.withProject(elem.value.project).traverse { v =>
```

Comment:
Doesn't feel great that this is optional - most of the concrete types for which we're querying elems have a project and they are mandatory in the scoped tables. I guess it's because `Envelope` was used for both global and scoped?

Reply:
It's confusing how the elem wouldn't already have the project set since it's coming from within it..?

Reply:
You should be able to populate this field from the original SQL query.
Yes, it was because `Envelope` and `Elem` were designed to work for both global and scoped entities.
Not sure if there is still a use case for global entities though now that we got rid of the sse events for them.